### PR TITLE
Operations history: output ≈ +- 0.01 $ if a sum is not zero but less …

### PR DESCRIPTION
**Pull request by keshan3262**
…than 1 cent by absolute value

### CODE REVIEW

File: src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx [hunk 1]
```diff
@@ -82,10 +79,11 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
...
           style={[
             styles.valueText,
-            conditionalStyle(amount.isPositive(), styles.positiveAmountText, styles.negativeAmountText)
+            conditionalStyle(amount > 0, styles.positiveAmountText, styles.negativeAmountText)
           ]}>
-          {amount.isPositive() ? '+ ' : '- '}
-          {amount.abs().toFixed()}
+          {Math.abs(amount) < 0.01 && '≈ '}
+          {amount > 0 ? '+ ' : '- '}
+          {roundFiat(new BigNumber(amount > 0 ? Math.max(amount, 0.01) : Math.min(amount, -0.01)).abs()).toFixed()}
```
Comment (unixvb) [inline src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx:86]: reuse 
`import { clamp } from 'lodash-es';`

File: src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx [hunk 2]
```diff
@@ -82,10 +79,11 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
           key={index}
           style={[
             styles.valueText,
-            conditionalStyle(amount.isPositive(), styles.positiveAmountText, styles.negativeAmountText)
+            conditionalStyle(amount > 0, styles.positiveAmountText, styles.negativeAmountText)
           ]}>
-          {amount.isPositive() ? '+ ' : '- '}
-          {amount.abs().toFixed()}
+          {Math.abs(amount) < 0.01 && '≈ '}
```
Comment (unixvb) [inline src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx:84]: create `minAmountValue` & `maxAmountValue` variables and use 
`import { inRange } from 'lodash-es';`

File: src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx [hunk 3]
```diff
@@ -62,10 +66,7 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
       }
     }
 
-    const positiveDollarSum = roundFiat(new BigNumber(positiveAmountSum));
-    const negativeDollarSum = roundFiat(new BigNumber(negativeAmountSum));
-
-    return { amounts, dollarSums: [positiveDollarSum, negativeDollarSum].filter(sum => !sum.eq(0)) };
+    return { amounts, dollarSums: [positiveAmountSum, negativeAmountSum].filter(sum => sum !== 0) };
```
Comment (unixvb) [inline src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx:69]: `dollarSums` is not used.  I'm I right?

Comment (keshan3262) [reply src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx:69]: You're wrong https://github.com/madfish-solutions/templewallet-mobile/blob/461c46734c3b9671af4e9b3e1f275662d5720ff8/src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx#L81

**Review by unixvb — APPROVED** [review]

**Review by andrewkryshtal — APPROVED** [review]

**Pull request merged**